modules/manually-rotating-cloud-creds.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
```

Comment:
This looks fine. I'm just throwing this out there since the thought came to me after reading these docs... Another alternative for verification is that you can view/save the contents of the Secrets before deleting them, and the compare with the newly created Secrets. The contents of the Secret will have changed to reflect the fact that CCO had to go mint new credentials and then saved them to the Secret.

Reply:
Frankly that sounds much cleaner in that 1) it is simple, and 2) doesn't require logging into the cloud provider. I'll save these steps off somewhere in case there's a request specifically for provider-side verification, but will publish with the process you suggest here instead. Thanks!